plenigo_plugin/PlenigoContentManager.php

```diff
@@ -553,12 +569,13 @@ public function hasPreventTag() {
         return $res;
     }
 
-    private function getCategoryChildren($id) {
+    private function getCategoryChildren($id)
+    {
 
         $categories = array();
         $categories[$id] = $id;
 
-        $cats = get_categories( array( 'parent' => $id ));
+        $cats = get_categories(array('parent' => $id));
```

Comment:
This is OK, but perhaps it can be changed to `$cats = get_categories(['parent' => $id]);`